packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -870,19 +870,19 @@ class DefaultMaterialLocalizations implements MaterialLocalizations {
   String get continueButtonLabel => 'CONTINUE';
 
   @override
-  String get copyButtonLabel => 'COPY';
+  String get copyButtonLabel => 'Copy';
 
   @override
-  String get cutButtonLabel => 'CUT';
+  String get cutButtonLabel => 'Cut';
 
   @override
   String get okButtonLabel => 'OK';
 
   @override
-  String get pasteButtonLabel => 'PASTE';
+  String get pasteButtonLabel => 'Paste';
 
   @override
-  String get selectAllButtonLabel => 'SELECT ALL';
+  String get selectAllButtonLabel => 'Select all';
```

Comment:
How do I indicate that these strings have been changed and need to have their translations updated?

Reply:
Sounds like a @shihaohong or @HansMuller question? I'm interested in knowing the answer too.

Reply:
The instructions for adding the strings is a comment at the top of this file. I think there's a script that automatically does that in google3, so long as the default is modified here and in [material_en.arb](https://github.com/flutter/flutter/blob/master/packages/flutter_localizations/lib/src/l10n/material_en.arb). 

We should probably update those instructions to include a guide for how to modify existing strings.

Reply:
I did a quick investigation and whenever Flutter is rolled into google3, material_en.arb is the template that determines what new strings are added or if any existing strings were modified, so make sure to update those as well!

Reply:
Thanks for the help!  I just created a separate PR to add instructions for doing this based on my understanding.  More code review welcome :) https://github.com/flutter/flutter/pull/59273